### PR TITLE
Klipse integration

### PR DESCRIPTION
**Pull request by madstap**
Inserts klipse stuff from post/page.klipse

### CODE REVIEW

File: src/leiningen/new/cryogen/themes/blue/html/base.html [hunk 1]
```diff
@@ -95,5 +95,7 @@
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
 {% script "js/highlight.pack.js" %}
 <script>hljs.initHighlightingOnLoad();</script>
+{% if post.klipse %} {{post.klipse|safe}} {% endif %}
+{% if page.klipse %} {{page.klipse|safe}} {% endif %}
```
Comment (lacarmen) [inline src/leiningen/new/cryogen/themes/blue/html/base.html]: Code blocks looks a little strange when both hljs and klipse are applied in conjunction. Perhaps hljs should be disabled if klipse is enabled.

Comment (madstap) [inline src/leiningen/new/cryogen/themes/blue/html/base.html]: Using the blue theme I can't spot any difference, but it makes sense that there would be.

If it's totally disabled, then any code blocks not marked as evalable would have no highlighting. Probably not the most common use case, but still.

 Another way could be to use enlive to tag any blocks marked for eval with the class `nohilight`. What do you think?

(We can just count any strings with a leading dot that are values in the `:settings` map as evalable class names to make it future proof when new selectors are added.)

Comment (lacarmen) [inline src/leiningen/new/cryogen/themes/blue/html/base.html]: I think users can overcome this by adding their own css so let's leave it alone then. Trying to accommodate this use case sounds like it's more work than it's worth.

Comment (madstap) [inline src/leiningen/new/cryogen/themes/blue/html/base.html]: I made a new commit implementing this in cryogen-core

Comment (madstap) [inline src/leiningen/new/cryogen/themes/blue/html/base.html]: I didn't refresh this page so I didn't see your message, and that you merged my commits. Hehe. It wasn't a lot of work in any case, enlive is really nice. I'll open a new PR.

**Pull request merged**